### DISCUSSION TITLE
.ipynb files created by COLAB are not formatted correctly

### DISCUSSION THREAD

**Discussion by halfhoon** [community/community#108251] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/158256936/e78674d8-0ffe-4d4d-bb6b-eb7124babf94)

hello. I recently committed a file written in colab to GitHub, and I'm having an issue with neither indentation nor line breaks appearing correctly, as shown in the photo above.

I have the same problem even when I change browsers or use a different computer. 

I'm guessing it's something with the ipynb file (the .py file appears just fine), as the code in other repositories I've referenced looks the same, so how can I fix it?

**Comment by EllenB** [comment]
Hi,

I am facing the same issue.

**Reply by PawelJakubczyk** [reply]
Hello, I have the same problem and I can't find a solution. If you find something, please share it, thanks in advance

**Comment by zidnyyasrah** [comment]
Colab and GitHub might have different preferences for indentation (spaces vs. tabs).
